pilot/pkg/model/context.go

```diff
@@ -137,18 +137,30 @@ const (
 
 	// IngressKeyFilename is the ingress private key file name
 	IngressKeyFilename = "tls.key"
+
+	// ConfigPathDir config directory for storing envoy json config files and also core files
+	ConfigPathDir = "/etc/istio/proxy"
```

Comment:
maybe a link to what makes the cores end up in that directory would be helpful (and if someone changes it, a back link to here so they know something needs to change)

Reply:
yes, it's in the init container.

Reply:
thanks for the comment!

I hope the various helm and other templates all have the same value